scripts/install

```diff
@@ -259,10 +259,8 @@ ynh_multimedia_addaccess $app
 # Fix app ownerships & permissions
 chown -R $app:www-data "$install_dir"
 chown -R $app: "$data_dir"
-find $install_dir/ -type f -print0 | xargs -r0 chmod 0644
-find $install_dir/ -type d -print0 | xargs -r0 chmod 0755
-find $data_dir/data/ -type f -print0 | xargs -r0 chmod 0640
-find $data_dir/data/ -type d -print0 | xargs -r0 chmod 0750
+chmod -R a=r,u+w,a+X $install_dir/
```

Comment:
(Any reason we have to have `755` for the install_dir rather than `750` ? x_X)

Reply:
I don't know, but some lines after we put 750 on the dir (without recursive option) so i guess those files are protected anyway from others users...